frontend/widgets/CMSProductRecommendations/style.js

```diff
@@ -0,0 +1,38 @@
+import { css } from 'glamor';
+
+/**
+ * @param {string} CTABackgroundColor background-color
+ * @param {string} CTAColor color
+ * @return {StyleAttribute}
+ */
+const button = (CTABackgroundColor, CTAColor) => css({
```

Comment:
not sure if all the !importants are needed here, if yes we should maybe change the core/create a ticket for it